applications/main/nfc/plugins/supported_cards/csc.c

```diff
@@ -0,0 +1,203 @@
+/*
+* Parser for CSC Service Works Reloadable Cash Card (US)
+* Date created 2024/5/26
+* Zinong Li  
+* Discord  @torron0483 
+* Github   @zinongli
+*/
+
+#include "nfc_supported_card_plugin.h"
+#include <flipper_application.h>
+#include <nfc/protocols/mf_classic/mf_classic_poller_sync.h>
+#include <bit_lib.h>
+#include <furi/core/string.h>
+
+#include <nfc/nfc.h>
+#include <nfc/nfc_device.h>
+
+#define TAG "CSC"
+#define csc_MAGIC_NUMBER 0x21
+
+typedef struct {
+    uint64_t a;
+    uint64_t b;
+} MfClassicKeyPair;
+
+static const MfClassicKeyPair csc_1k_keys[] = {
+    {.a = 0xF329F7AAEDFF, .b = 0xffffffffffff},
+    {.a = 0xF329F7AAEDFF, .b = 0xffffffffffff},
+    {.a = 0xF329F7AAEDFF, .b = 0xffffffffffff},
+    {.a = 0xF329F7AAEDFF, .b = 0xffffffffffff},
+    {.a = 0xffffffffffff, .b = 0xffffffffffff},
+    {.a = 0xffffffffffff, .b = 0xffffffffffff},
+    {.a = 0xffffffffffff, .b = 0xffffffffffff},
+    {.a = 0xffffffffffff, .b = 0xffffffffffff},
```

Comment:
repetition should be fine to remove, also are the 0xffffffffffff real keys being used?